sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClientOptions.cs

```diff
@@ -32,7 +32,7 @@ public CallAutomationClientOptions(ServiceVersion version = LatestVersion)
         {
             ApiVersion = version switch
             {
-                ServiceVersion.V2023_03_06 => "2023-03-06",
+                ServiceVersion.V2023_03_06 => "2023-06-15-preview",
```

Comment:
I think all future version of our release should be backward compatibility without breaking change. we should keep the serviceversion V2023_03_06 and its value, but add a new service version for this beta. Then make the new version latest version

Reply:
done, thanks!